ArduPlane/mode_auto.cpp

```diff
@@ -159,3 +159,8 @@ bool ModeAuto::_pre_arm_checks(size_t buflen, char *buffer) const
     // Note that this bypasses the base class checks
     return true;
 }
+
+bool ModeAuto::is_landing() const
+{
+    return (plane.flight_stage == AP_FixedWing::FlightStage::LAND) || plane.mission.get_in_landing_sequence_flag();
```

Comment:
I'm not sure we should use `get_in_landing_sequence_flag` that can be set for log parts of missions that may not even have a actual landing at the end. 

I would be tempted to just put `plane.flight_stage == AP_FixedWing::FlightStage::LAND` in the base class method and forget about the auto override.

Reply:
I think it's worthwhile to keep in. Same applies to the q VTOL landing stuff. It's vague but it's showing intent

Reply:
At least the VTOL landing stuff will actually end in a landing eventually. `get_in_landing_sequence_flag` just means we have been past a `DO_LAND_START`. The VTOL landing stuff also lands at the current location, the lading sequence can start a long way away from where the actual landing is going to happen.

Reply:
For the sake of getting the rest in, I'll remove the mission check but I'd like to discuss adding it on a follow up PR

Reply:
removed, only the stage is used now